index.d.ts

```diff
@@ -2,13 +2,13 @@ export function parse(connectionString: string): ConnectionOptions;
 
 export interface ConnectionOptions {
   host: string | null;
-  password: string | null;
-  user: string | null;
-  port: number | null;
-  database: string | null;
-  client_encoding: string | null;
-  ssl: boolean | null;
+  password?: string;
+  user?: string;
+  port: string | null;
+  database: string | null | undefined;
```

Comment:
An optional parameter essentially means `| undefined`. Should be `database?: string | null;`